document/AnnexG-Examples.mdk

```diff
@@ -0,0 +1,466 @@
+# Annex G (informative) Examples { - #Annex-G }
+
+This Annex contains examples of complete, valid SARIF files,
+to complement the fragments shown in examples throughout this document.
+
+## Minimal valid SARIF file resulting from a scan { - #AnnexG-Minimal }
+
+This is a minimal valid SARIF file for the case where the analysis tool was run with the
+intent of scanning files and producing results (see &sect;[#run-results]).
+The file contains only those elements required by the specification
+(that is, those elements which the specification states "shall" be present).
+
+The file contains a single `run` object (&sect;[#run]) with an empty `results` array (&sect;[#run-results]),
+as would happen if the tool detected no issues in any of the files it scanned.
+
+```
+{
```

Comment:
The "Minimal recommended" file below is quite short and contains a single `result`, so I think that serves the purpose.

---
In reply to: [72474811](https://github.com/sarif-standard/sarif-spec/pull/248#discussion_r72474811) [](ancestors = 72474811)